### PR TITLE
PTcasestudy2

### PR DESCRIPTION
**Pull request by mythod**

### CODE REVIEW

**Comment by mbMosman** [comment]
Please update Git Hub with your name to facilitate grading.

File: activities.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+
```
Comment (mbMosman) [inline activities.html]: Missing your `<head>` tag.

File: activities.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+<title>Pacific Trails Resort :: Activities</title>
+    
+<link rel="stylesheet" href="pacific.css">
+<header><h1>Pacific Trails Resort</h1></header>
```
Comment (mbMosman) [inline activities.html]: Missing your `<body>` tag.

File: activities.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+</nav>
+<br>
+<img src="images/coast.jpg" height="250" width="320" alt="A beautiful caost line">
+
+    
+
+<h2>Activities at Pacific Trails</h2>
+
+    <div id="wrapper">
```
Comment (mbMosman) [inline activities.html]: The wrapper div should enclose all of the body contents - the header, nav, main content, and the footer.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+<a href="index.html">Home</a> &nbsp;<a href="yurts.html">Yurts</a> &nbsp;<a href="activities.html">Activities</a> &nbsp;<a href="reservations.html">Reservations</a> &nbsp;   
+    
+</nav>
+<br>
+
+<img src="images/coast.jpg" height="250" width="320" alt="A beautiful coast line">
+
+<h2>Enjoy Nature in Luxury</h2>
+
+<div id="wrapper">
```
Comment (mbMosman) [inline index.html]: The wrapper div should enclose all of the body contents - the header, nav, main content, and the footer.  This should be the `<main>` tag.

File: yurts.html [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+<a href="index.html">Home</a> &nbsp;
+<a href="yurts.html">Yurts</a> &nbsp;
+<a href="activities.html">Activities</a> &nbsp;
+<a href="reservations.html">Reservations</a> &nbsp;
+
+</nav>
+<br>
+<img src="images/coast.jpg" height="250" width="320" alt="A beautiful coast line">
+
+    <div id="wrapper">
```
Comment (mbMosman) [inline yurts.html]: The wrapper div should enclose all of the body contents - the header, nav, main content, and the footer.  This should be the `<main>` tag.

File: activities.html [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+<dt><strong>What should I bring?</strong></dt>
+<dd>Bring a sense of adventure and some time to relax! Most guests also pack comfortable walking shoes and a plan to dress for changing weather with layers of clothing.</dd>
+    </dl>
+    </div>
+
+        
+    
+<br>
+<footer>
+<small>Copyright &copy; 2016 Pacific Trails Resort</small>
```
Comment (mbMosman) [inline activities.html]: You can remove the `<small>`tags.  This is done w/ CSS now.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+
+    <div id="contact">
+    <span class="resort">Pacific Trails Resort</span>
+    <br>12010 Pacific Trails Road
+    <br>Zephyr, CA 95555
+    <br>888-555-5555
+    </div>
+        <footer>
+            <small>Copyright &copy; 2016 Pacific Trails Resort</small><br>
+                <small><a href="mailto:cfristed@pacifictrails.com">cfristed@pacifictrails.com</a></small>
```
Comment (mbMosman) [inline index.html]: You can remove the `<small>`tags.  This is done w/ CSS now.

File: yurts.html [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+
+    </div>
+
+
+    
+<br>    
+<footer>
+    <small>Copyright &copy; 2016 Pacific Trails Resort</small><br>
+    <small><a href="mailto@cfristed@pacifictrails.com">cfristed@pacifictrails.com</a></small>
+</footer>
```
Comment (mbMosman) [inline yurts.html]: You can remove the `<small>`tags.  This is done w/ CSS now.

File: yurts.html [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+<br>
+<img src="images/coast.jpg" height="250" width="320" alt="A beautiful coast line">
+
+    <div id="wrapper">
+<h2>The Yurts at Pacific Trails</h2>
+
+
+
+<br>
+        <dl>
```
Comment (mbMosman) [inline yurts.html]: The instructions were to use `<h3>` and `<p>` tag for this page.  Be sure to use the web site instructions not the textbook.

**Comment by mbMosman** [comment]
The issues from part 1 were not resolved and that is compounding the problem here for part 2.  Please reach out to me in chat or email if you need help.  If the issues continue through part 3, you risk failing the class due to low scores.

**Comment by mythod** [comment]
The only reason some of the tags were missing was the validate told me to
remove them...I have put them back in and made all the corrections, the
third part should be what you are looking for...if I have any further
questions I'll be sure to hit you up.

Thanks,
Chris Fristed-BP Thursday night html

**Pull request closed without merging**